vault/logical_system.go

```diff
@@ -1342,6 +1343,16 @@ func (b *SystemBackend) handleTuneWriteCommon(ctx context.Context, path string,
 				resp = &logical.Response{}
 				resp.AddWarning(fmt.Sprintf("Upgrading mount from version %d to version %d. This mount will be unavailable for a brief period and will resume service shortly.", meVersion, optVersion))
 			}
+		} else {
+			// if version is not included in the
```

Comment:
This is a nit-pick, but the text width on this comment block seems off. It should be 80 chars no matter the indentation.

Reply:
A good nit though, not sure how I let that by I'm normally pretty on-top of things like that 😄 
Fixed in 8f19ec962